NuGet.Config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetvnext/api/v2" />
-    <add key="NuGet" value="https://nuget.org/api/v2/" />
+    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetlitedev/api/v2" />
+    <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
```

Comment:
there's still something odd with your rebase if you are including changes in this file.